### PR TITLE
Added case studies

### PR DESCRIPTION
**Pull request by szczepanbarszczowski**

### CODE REVIEW

File: src/routes/CaseStudies/index.js [hunk 1]
```diff
@@ -11,8 +24,133 @@ export const CaseStudies = () => (
...
+    <div className={c.caseStudies}>
+      <div className={c.heading}>
+        <h1 className={c.title}>
+          A brief selection of<br />our <span className={c.titleBold}>case studies</span>
+        </h1>
+      </div>
+
+      <div className={cn(c.firstSection, c.section)}>
+        <div className={cn(c.firstCol, c.colBig)}>
+          <div>
```
Comment (ArekRado) [inline src/routes/CaseStudies/index.js]: why empty div?

File: src/routes/CaseStudies/index.js [hunk 2]
```diff
@@ -11,8 +24,133 @@ export const CaseStudies = () => (
...
+            <img src={diorSection} alt="Dior" className={c.img} />
+          </div>
+        </div>
+
+        <div className={cn(c.secondCol, c.colSmall)}>
+          <CaseStudyBrief
+            text={`Worldwide clients\nmanagement`}
```
Comment (ArekRado) [inline src/routes/CaseStudies/index.js]: use `" "` for strings

Comment (szczepanbarszczowski) [reply src/routes/CaseStudies/index.js]: I need line break

File: src/routes/CaseStudies/index.js (hunk 1 above)
Comment (szczepanbarszczowski) [reply src/routes/CaseStudies/index.js]: It's hack for aspect on images

**Pull request merged**